src/test/java/com/box/sdk/BoxAPIResponseTest.java

```diff
@@ -85,4 +144,18 @@ public void testJsonResponseToString() {
                 + "Body:\n{\"foo\":\"bar\"}")
         );
     }
+
+    @Test
+    public void testLargeBinaryResponseContentLength() {
+        long contentLength = 2352815193L;
```

Comment:
What is this magic number? You want it to be larger than MAX int? Some random value?

Reply:
This is a number bigger than Max Int and which is customer having issue with, can be also considered as a random number :/